data/quarg/quarg missions.txt

```diff
@@ -113,3 +113,134 @@ mission "First Contact: Kuwaru Efreti"
+			choice
+				`	"Are those other Captains also doing this? What do you need so much cargo for?"`
+				`	"I'm not looking for any more cargo jobs right now, sorry."`
+					decline
+			`	"Many have agreed to help us already, but we still need many more. Those that have agreed to help are headed all to different portions of human space, to safely bring us the materials we have ordered.`
+			`	"Our ringworld, under construction in the nearby system, <system>, is to have a special open section for humans to visit soon. We wish to make the internal style and decoration of that section closer to human standards, so we are asking every capable captain to help. You are one such captain, are you willing to help us?"`
+			choice
+				`	"Alright, I'm interested. Where should I go to pick up the cargo?"`
+				`	"Sorry, I'm not looking for any cargo job right now, I was just curious."`
+					decline
+			`	It fiddles with the data pad a bit more, then says, "You are to pick up <cargo> on <stopovers>, then deliver that cargo to <destination>. You will be paid <payment> for the service. Be well, human."`
+			`	It gestures a bow to you, then leaves to speak with another merchant captain that got curious as well.`
+				accept
+	on stopover
+		dialog `The <cargo> is loaded into your ship under strict supervision, each crate carefully accounted for.`
+	on complete
+		payment 953000
+		conversation
+			`Much like your own ship, several other human freighters and convoys are arriving, and the cargo being meticulously checked by the Quarg, as they use their own machinery to unload what must be tens of thousands of tons from all ships.`
+			`	About halfway through your own cargo, a Quarg comes up to you and hands you <payment>.`
+
+
+
+mission "Ask Quarg About Coalition Early"
+	landing
+	to offer
+		has "First Contact: Quarg: offered"
+		has "Coalition: First Contact: done"
+		not "assisted heliarch"
+		not "Heliarch Recon 1: offered"
+		not "Resistance: Introductions: offered"
+	source
+		attributes "quarg" "station"
+	on offer
+		conversation
+			`You've met the Coalition, and the Heliarchs that rule their space from within the ringworlds the Quarg built there, and that claim that they "defeated the Quarg oppressors". Would you like to look for some Quarg here and ask for their side of the story?`
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					decline
+			`	You stop the first Quarg that passes by you, and ask if they could tell their side of the story about the Coalition.`
+			`	"Ah, so you have encountered the foolish Coalition," it begins. "They once were three promising species, before they succumbed to the unwise desire for usurpation of the great deeds of others.`
+			`	"Not that all of them are petty thieves, but sadly the civilization they have become is now ruled by a maniacal military, as I am sure you have seen, having spoken with those wretched Heliarchs"`
```

Comment:
perhaps change 
>"maniacal military"

into "fanatical military" , as it would match the end of the conversation more (which says "fanatical")